frontend/src/AnalyticsPage.tsx

```diff
@@ -0,0 +1,120 @@
+                        onRemove={this.onSelectionChange.bind(this)}
+                        displayValue="label"
+                        closeIcon="cancel"
+                        groupBy="type"
+                    />
+                    {SnippetManager.createInfoWithoutTooltip("https://confluence.supersocial.cloud/display/SP/Filters+on+the+Analytics+Dashboard", "Hint: If you select all filters, the result is the same as if no filter was selected at all. The filter are combined with a logical or. More information are available ", "hint")}
+                </div>
+                <PostAnalyticsTile analyticsNumber={post} label="# Impressions" eventBus={this.props.eventBus} keyVal="impressions" type={AnalyticsType.POST} background="binoculars" />
+                <PostAnalyticsTile analyticsNumber={post} label="# Comments" eventBus={this.props.eventBus} keyVal="comments" type={AnalyticsType.POST} background="comment" />
+                <PostAnalyticsTile analyticsNumber={post} label="# Likes" eventBus={this.props.eventBus} keyVal="likes" type={AnalyticsType.POST} background="like" />
+                <PostAnalyticsTile analyticsNumber={post} label="# Shares" eventBus={this.props.eventBus} keyVal="shares" type={AnalyticsType.POST} background="share" />
+                <PostAnalyticsTile analyticsNumber={post} label="# Followers" eventBus={this.props.eventBus} keyVal="acc_followers" type={AnalyticsType.ACCOUNT} background="follower" />
+                <PostAnalyticsTile analyticsNumber={post} label="# Posts" eventBus={this.props.eventBus} keyVal="acc_posts" type={AnalyticsType.ACCOUNT} background="post" />
+            </div>
+        );
+    }
+
+    /***Called by the Multiselect UI element. Synchronizes the multiselect state to the component state. */
+    private onSelectionChange(value: MultiSelectValue[]) {
+        let accounts = value.filter(v => v.type === "Accounts").map(v => this.state.accounts.find(sa => v.value === sa.id));
+        let posts = [];//value.filter(v => v.type === "Posts").map(v => this.state.posts.find(sa => v.value == sa.id));
+        this.updateData(EventBusEventType.REFRESH_POST_ANALYTICS_REQ, "", accounts as AccountDataSource[], posts as PostDataSource[]);
+        this.updateData(EventBusEventType.REFRESH_ACCOUNT_ANALYTICS_REQ, "", accounts as AccountDataSource[]);
+    }
+
+    /** Called by the event handler when new data are to be fetched. */
+    private updateData(eventType: EventBusEventType, relId: string, selectedAccounts?: AccountDataSource[], selectedPosts?: PostDataSource[]) {
+        let typedPath = eventType === EventBusEventType.REFRESH_ACCOUNT_ANALYTICS_REQ ? "account" : "post";
+        let params = (selectedPosts ? selectedPosts.map(p => p ? "&posts=" + p.id : "").join("") : "") + (selectedAccounts ? selectedAccounts.map(a => a ? "&accounts=" + a.id : "").join("") : "");
+        fetch(DeploymentManager.getUrl() + "api/analytics/" + typedPath + "/" + relId + "?query=" + params, {
```

Comment:
Could be simplified to:
```suggestion
        fetch(`${DeploymentManager.getUrl()}api/analytics/${typedPath}/${relId}?query=${params}`, {
```

Reply:
Implemented for all URL patterns for fetch calls with commit f7e5fa7cd61f5302ac85a0c2e5ee6212b45a39c9